versions.txt

```diff
@@ -0,0 +1 @@
+required_glib_version=2.46
```

Comment:
It's probably also worth adding the following from `.travis-setup.sh`:

```
json_glib_version
check_version
```

Also, `versions.txt` will need to be added to `EXTRA_DIST` in `Makefile.am` to ensure it gets included in any release tarballs and can thus still be sourced by `configure.ac`.